docs/features/files.md

```diff
@@ -18,3 +18,19 @@ new GenericContainer(...)
                                       "/etc/redis.conf",
                                       BindMode.READ_ONLY)
 ```
+
+## Copying files to and from containers
+
+Files can be copied into the container before startup, or can be copied from the container after the container has started.
+
+### Copying to a container before startup
+
+<!--codeinclude-->
+[Copying files to a container](../../core/src/test/java/org/testcontainers/junit/CopyFileToContainerTest.java) inside_block:copyToContainer
```

Comment:
For some reason the whole file got included:
https://deploy-preview-4661--testcontainers.netlify.app/features/files/